src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -181,9 +181,22 @@ export default class ItemDetailPage extends React.PureComponent<Props, State> {
                       </span>
                       <div className="urn-actions">
                         {isThirdParty(item.urn) ? (
-                          <span className="link" onClick={this.handleEditURN}>
-                            {t('item.edit_urn')}
-                          </span>
+                          <Popup
+                            content={t('collection_item.cannot_edit_urn')}
+                            disabled={false}
+                            position="top center"
+                            trigger={
+                              <div>
```

Comment:
The button seems to have a left padding that pushes it a bit to the right and, as it doesn't have the button borders and such, it is seen a bit as uncentered:
![Screen Shot 2023-01-11 at 13 51 03](https://user-images.githubusercontent.com/1120791/211867160-5a21cadb-fb0d-48c7-b1f8-77f4d110ee7a.png)

Reply:
good catch, padding removed!